compute/etc/sql_exporter/autovacuum_stats.sql

```diff
@@ -0,0 +1,19 @@
+WITH dbs AS (
+  SELECT datname 
+  FROM pg_database 
+  WHERE datconnlimit != -2
+    AND datname NOT IN ('postgres')
+    AND NOT datistemplate
+)
+SELECT 
+  d.datname as dbname,
+  t.autovacuum_count as autovacuum_count
+FROM dbs d,
+LATERAL (
+  SELECT * FROM dblink(
+    'dbname=' || d.datname || ' user=' || current_user || ' connect_timeout=5',
```

Comment:
Please make sure you correctly escape the database and user parameters of this connection string.